Composer/packages/client/src/pages/publish/Publish.tsx

```diff
@@ -45,6 +50,13 @@ import {
   deleteNotificationInterval,
 } from './publishPageUtils';
 
+const SKILL_PUBLISH_STATUS = {
+  INITIAL: 'inital',
```

Comment:
I'm assuming this is not used in UI at all. If it is we need to use formatMessage.

Reply:
Sure. It is not used in ui